src/pages/RepoPage/CoverageTab/OverviewTab/subroute/Sunburst/Sunburst.jsx

```diff
@@ -23,6 +23,13 @@ function Sunburst() {
 
   const breadcrumbPaths = useConvertD3ToBreadcrumbs(currentPath)
 
+  let pathsToDisplay
+  if (breadcrumbPaths.length <= 1) {
+    pathsToDisplay = breadcrumbPaths
+  } else {
+    pathsToDisplay = [breadcrumbPaths[0], { text: '...' }]
```

Comment:
Can we extend this to more than just one level? Something like 2 or 3 so it matches the depth you can travel while looking at the sunburst? I find it pretty difficult to get any context as to "where" i am in the file structure with just the current level.

Reply:
Could also poke the design folks, and get their input on this.

Reply:
Thanks, I'll ping design. This is just a quick and dirty. Really we have too much information trying to be displayed in too little of space. I added some screenshots to the PR description to show what it's like. If it's any consolation, there's the hover tooltip on the sunburst - I can add one to the breadcrumbs as well perhaps.
If we want to do this for real a reusable truncated breadcrumbs component would be ideal imo

Reply:
here's a note from talking to adalene - sounds like we can get some design love on this in general to make it better for users
![Screenshot 2024-10-22 at 1 42 13 PM](https://github.com/user-attachments/assets/3ed8e855-da13-4754-be8e-6949c058e8f9)